module/parser/src/iosMain/kotlin/org/cru/godtools/tool/model/Content.kt

```diff
@@ -0,0 +1,7 @@
+package org.cru.godtools.tool.model
+
+import org.cru.godtools.tool.state.State
+import org.cru.godtools.tool.util.watch
+
+fun Content.watchIsGone(state: State, block: (Boolean) -> Unit) = isGoneFlow(state).watch(block)
+fun Content.watchIsInvisible(state: State, block: (Boolean) -> Unit) = isInvisibleFlow(state).watch(block)
```

Comment:
I'm not sure if you will need these values combined to a single `watch` method, we can sort this out if you do.